### PR TITLE
Group creation members example

### PR DESCRIPTION
**Pull request by Jackson-Woods**
Showed how to add members immediately upon group creation in second request example (v1 and beta).

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[41dfcad](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/41dfcadfe07cecab1fa6b85bce89424cd95685ed)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[41dfcad](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/41dfcadfe07cecab1fa6b85bce89424cd95685ed)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[41dfcad](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/41dfcadfe07cecab1fa6b85bce89424cd95685ed)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/beta/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C19%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901190059258251-3824%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=hML6LdP1RbjB1lprNbIjPyxjAEHv4k7grLhB6KkFOuc%3D&st=2019-01-19T00%3A57%3A44Z&se=2019-02-19T01%3A02%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/group-post-groups.md [hunk 1]
```diff
@@ -143,17 +143,17 @@ Content-type: application/json
...
-The second example request creates an Office 365 group with an owner specified.
+The second example request creates an Office 365 group with an owner and members specified.
 <!-- {
   "blockType": "request",
-  "name": "create_group_with_owner"
+  "name": "create_prepopulated_group"
 }-->
 ```http
-POST https://graph.microsoft.com/beta/groups
+POST https://graph.microsoft.com/v1.0/groups
```
Comment (Lauragra) [inline api-reference/beta/api/group-post-groups.md]: This is a beta topic; did you intend to change this to v1.0?

Comment (Jackson-Woods) [reply api-reference/beta/api/group-post-groups.md]: Thanks for the catch - this is a mistake.
```suggestion
POST https://graph.microsoft.com/beta/groups
```

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[91f216a](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/91f216ab7cf1dab161050247513a578d9df1467d)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[91f216a](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/91f216ab7cf1dab161050247513a578d9df1467d)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[91f216a](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/91f216ab7cf1dab161050247513a578d9df1467d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/beta/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C22%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901220057571014-3824%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dbkAbs0f8%2FJtXbqA4eGehx1Oo22DUnmCTN33szNkDVY%3D&st=2019-01-22T00%3A56%3A12Z&se=2019-02-22T01%3A01%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8366eb6](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/8366eb6c3059edb70a04367aaa343a3e72f19645)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8366eb6](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/8366eb6c3059edb70a04367aaa343a3e72f19645)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8366eb6](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/8366eb6c3059edb70a04367aaa343a3e72f19645)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/beta/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C22%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901220102324957-3824%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4%2FDCqagsZuCkZe6JS6aCyMqEdRD8cte%2B8elGR2dRPSA%3D&st=2019-01-22T01%3A00%3A46Z&se=2019-02-22T01%3A05%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d83e708](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/d83e70839e2020791efeae22d8bc9d334e70f07c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d83e708](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/d83e70839e2020791efeae22d8bc9d334e70f07c)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[d83e708](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3824/commits/d83e70839e2020791efeae22d8bc9d334e70f07c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/beta/api/group-post-groups.md) | :white_check_mark:Succeeded | |
[api-reference/v1.0/api/group-post-groups.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/jsw-groups-members-example/api-reference/v1.0/api/group-post-groups.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C23%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901230256198097-3824%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=rAhZIdUyBay7FhX5w9KqeQTbh5e7r%2FnYl9x%2BGTBn6N8%3D&st=2019-01-23T02%3A54%3A49Z&se=2019-02-23T02%3A59%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**